README.md

```diff
@@ -37,10 +37,9 @@ import { StakingClient } from "@coinbase/staking-client-library-ts";
 const client = new StakingClient();
 
 client.Ethereum.stake(
-  'your-project-id', // replace with your project id
   'holesky',
   true,
-  'your-wallet-address', // replace with your wallet address
+  '0xdb816889F2a7362EF242E5a717dfD5B38Ae849FE', // replace with your wallet address
```

Comment:
can we roll this back ? or was this intentional ?

Reply:
Per offline discussion: Will keep in for simplicity of the example 😄.